src/components/StateEligibility.js

```diff
@@ -48,71 +75,44 @@ export default function StateEligibility() {
-                                    rel="noreferrer"
-                                    target="_blank"
-                                >
-                                    Individuals with two or more of certain
-                                    medical conditions
-                                </a>
-                            </div>
-                        </EligibilityGroupItem>
-                        <EligibilityGroupItem>
-                            <a
-                                href="https://www.mass.gov/info-details/covid-19-vaccinations-for-senior-housing-settings"
-                                rel="noreferrer"
-                                target="_blank"
-                            >
-                                Residents and staff of low-income and affordable
-                                senior housing
-                            </a>
-                        </EligibilityGroupItem>
-                        <EligibilityGroupItem>
-                            <a
-                                href="https://www.mass.gov/info-details/covid-19-vaccinations-for-people-ages-75-and-older"
-                                rel="noreferrer"
-                                target="_blank"
-                            >
-                                Individuals age 75 and older
-                            </a>
-                        </EligibilityGroupItem>
-                        <EligibilityGroupItem>
-                            <a
-                                href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases#phase-1-"
-                                rel="noreferrer"
-                                target="_blank"
-                            >
-                                People in Phase 1 (healthcare, nursing homes,
-                                etc.)
-                            </a>
-                        </EligibilityGroupItem>
-                        {/* TODO - remove the following div after March 11, and update link to be appropriate link from https://www.mass.gov/covid-19-vaccine*/}
-                        <br></br>
-                        <br></br>
-                        <div>Eligible to sign up starting March 11: </div>
-                        <br></br>
-                        <EligibilityGroupItem>
-                            <a
-                                href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases"
-                                rel="noreferrer"
-                                target="_blank"
-                            >
-                                Pre-K to 12 teachers and staff, early educators,
-                                and child care workers
-                            </a>
-                        </EligibilityGroupItem>
+                        {criteriaGroups.map((g, idx) => {
```

Comment:
nothing to change right now but I prefer using slightly more descriptive variable names, unless it's like a two-liner or something.

Reply:
You're quite right it started life as a 1-liner but didn't  get revised.
Fixed in https://github.com/johnhawkinson/macovidvaccines.com/commit/ebd313eec28d349812283ea1645028941263b1ad
which will be overwritten in the PR by the force-push for clean history for a merge commit.